src/main/java/com/google/api/generator/spring/composer/SpringAutoConfigClassComposer.java

```diff
@@ -155,41 +158,91 @@ private static List<Statement> createMemberVariables(
 
     Statement loggerStatement =
         LoggerUtils.getLoggerDeclarationExpr(serviceName + "AutoConfig", types);
-    return Arrays.asList(clientPropertiesStatement, loggerStatement);
+    Statement globalPropertiesStatement =
+        GlobalPropertiesUtils.getGlobalPropertiesDeclaration(types);
+    return Arrays.asList(clientPropertiesStatement, globalPropertiesStatement, loggerStatement);
   }
 
   private static MethodDefinition createConstructor(
       String serviceName, String className, Map<String, TypeNode> types, Expr thisExpr) {
-    VariableExpr propertiesVarExpr =
+    /// constructor
+    // VariableExpr credentialsProviderBuilderVarExpr =
+    //     VariableExpr.withVariable(
+    //         Variable.builder()
+    //             .setName("coreCredentialsProvider")
+    //             .setType(types.get("CredentialsProvider"))
+    //             .build());
+    //
+    // VariableExpr coreProjectIdProviderVarExpr =
+    //     VariableExpr.withVariable(
+    //         Variable.builder()
+    //             .setName("coreProjectIdProvider")
+    //             .setType(types.get("GcpProjectIdProvider"))
+    //             .build());
```

Comment:
The commented out code can be deleted. It probably sneaked in from rebase/merge.

Reply:
Removed!